OneCommunityGlobal/HighestGoodNetworkApp
# Ziming add google doc icon

# Description
(PRIORITY High) Jae: Add Google Doc icon and link to people in Tasks Tab (WIP Ziming)

Should be the same icon and link from the Reports>Weekly Summaries Reports page
Should still show gray if no Google Doc link present
Should, in this case, not be visible to Volunteers. Volunteers can see the icon if they have access to the Weekly Summaries Reports page. It's very important that that not change. 

## Related PRS (if any):
no

## Main changes explained:
- add google doc icon in `dashboard` task list
- google doc icon  show gray if no Google Doc link present
- Volunteers can see the icon if they have access to the Weekly Summaries Reports page. and can click the icon to see google doc.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
4. log as user
5. go to dashboard→ Tasks→ task
6. verify google doc shown in right color and can jump to google doc link correctly.

## Screenshots or videos of changes:
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/65f4afbc-1829-47d1-acff-7236f516973a
